js/util.js

```diff
@@ -38,4 +38,12 @@ const removeClass = (element, classToRemove) => {
 
 const isEscapeKey = (evt) => evt.key === 'Escape';
 
-export {getRandomInteger, getRandomFloat, getRandomArrayElement, getRandomArray, addClass, removeClass, isEscapeKey};
+const debounce = (callback, timeoutDelay) => {
```

Comment:
```suggestion
const debounce = (callback, timeoutDelay = 300) => {
```
второй параметр можно сделать со значекнием поумолчанию